build/tools/certgen.sh

```diff
@@ -71,6 +71,19 @@ stream() {
         SUBJECTALTNAME="${SUBJECTALTNAME}IP.${index}:${ip}"
     done
 
+
+    if [ -n "${CLOUDCOREDOMAIN}" ]; then
+        index=0
+        SUBJECTALTNAME="subjectAltName = "
```

Comment:
Looks like `SUBJECTALTNAME` will be reset here, is it possible to include both IPS and Domain? @hackers365